tests/testthat/test-gather.R

```diff
@@ -134,13 +134,11 @@ test_that("gather throws error for weird objects", {
   df <- data.frame(y = 1)
   df$x <- expression(x)
 
-  expect_snapshot(error = TRUE, cnd_class = TRUE, {
-    gather(d, key, val, -x)
+  # Can't use snapshot, it changes between versions of R
+  expect_error(gather(d, key, val, -x), "not found")
```

Comment:
```r
expect_error(gather(d, key, val, -x))
```

I think even this would be okay

Reply:
Makes sense! Should I remove the comment above?

Reply:
I think the comment is nice because i probably would wonder why we aren't snapshotting this one since we do snapshot the other!